agent/stats/utils.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package stats
+
+import (
+	"math"
+	"regexp"
+	"time"
+
+	"github.com/docker/libcontainer"
+)
+
+// networkStatsErrorPattern defines the pattern that is used to evaluate
+// if there's an error reading network stats.
+const networkStatsErrorPattern = "open /sys/class/net/veth.*: no such file or directory"
+
+// NaN32 returns a 32bit NaN.
+func NaN32() float32 {
+	return (float32)(math.NaN())
+}
+
+// ToContainerStats returns a new object of the ContainerStats object from libcontainer stats.
+func ToContainerStats(containerStats libcontainer.ContainerStats) *ContainerStats {
```

Comment:
Is this ever called outside the `stats` package?  If not, I'd prefer that it be made private.